### PR TITLE
Final tweaks

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

File: spec/semantics.html [hunk 1]
```diff
@@ -550,7 +550,7 @@
         1. Let _ye_ be _y_'s _endIndex_.
         1. <ins>If _direction_ is equal +1, then</ins>
           1. Let _s_ be a fresh List whose characters are the characters of _Input_ at indices _xe_ (inclusive) through _ye_ (exclusive).
-        1. <ins>Else, _direction_ is equal to -1.</ins>
+        1. <ins>Else, let _direction_ is equal to -1.</ins>
```
Comment (littledan) [inline spec/semantics.html]: I don't understand this insertion; I guess we could write "Assert" here.

File: spec/semantics.html [hunk 2]
```diff
@@ -550,7 +550,7 @@
         1. Let _ye_ be _y_'s _endIndex_.
         1. <ins>If _direction_ is equal +1, then</ins>
```
Comment (claudepache) [inline spec/semantics.html]: “is equal <ins>to</ins> +1”

File: spec/semantics.html (hunk 1 above)
Comment (claudepache) [reply spec/semantics.html]: Yeah, the addition of “let” is just plain wrong; it’s an assertion.

The following wording is sometimes used in ECMA262:
> 1. Else _x_ is "bar",
>     1. ...

(note the punctuation: no comma after “Else”, a comma at the end of line) in order to mean:

> 1. Else, 
>     1. Assert: _x_ is "bar".
>     1. ....

We should do the same: change every occurence of:
> 1. Else, _direction_ is equal to -1.

 into either:

> 1. Else _direction_ is equal to -1,

or:

> 1. Else,
>     1. Assert:  _direction_ is equal to -1.

**Review by claudepache — CHANGES_REQUESTED** [review]

File: spec/semantics.html (hunk 1 above)
Comment (mathiasbynens) [reply spec/semantics.html]: Yeah, I misunderstood.

Comment (mathiasbynens) [reply spec/semantics.html]: I like the last form in @claudepache’s comment — that makes this very explicit (and would’ve prevented my confusion).

**Comment by littledan** [comment]
Should this be merged?

**Comment by mathiasbynens** [comment]
@littledan That’s up to you, but the upstream PR includes these changes.

**Pull request merged**